src/hooks/useConnectedAddress.ts

```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from 'react';
 
 import { useWeb3React } from 'hooks/useWeb3React';
 
-export default function useConnectedAddress() {
+export function useConnectedAddress() {
```

Comment:
was there a reason for this change?

Reply:
to be able to import useConnectedAddress like this 
`import { useConnectedAddress } from 'hooks/useConnectedAddress';`
I had troubles mocking it in unit test and was getting this error:
`Error: Uncaught [TypeError: (0 , _useConnectedAddress.default) is not a function]`

Reply:
hmm... could you try:

```js
jest.mock('hooks/useConnectedAddress', () => {
  return {
    default: () => 'abc',
  };
});
```

Reply:
unfortunately it gave the same error as before. I reverted the changes and used another fix please check it

Reply:
sounds good